cpp/velox/memory/VeloxMemoryPool.h

```diff
@@ -21,12 +21,9 @@
 #include "velox/common/memory/Memory.h"
 
 namespace gluten {
-class WrappedVeloxMemoryPool;
 
-std::shared_ptr<facebook::velox::memory::MemoryPool> asWrappedVeloxAggregateMemoryPool(MemoryAllocator* allocator);
+std::shared_ptr<facebook::velox::memory::MemoryPool> asAggregateVeloxMemoryPool(MemoryAllocator* allocator);
 
-std::shared_ptr<facebook::velox::memory::MemoryPool> getDefaultVeloxAggregateMemoryPool();
```

Comment:
We should keep `defaultAggregateVeloxMemoryPool` to use in cpp test, use all the same name leaf pool is hard to debug.

Reply:
this function is not used by any other functions defined in the other files

Reply:
If no usages of the code I think it's fine to remove them. Feel free to add it back once it gets used again.